test/report/test-report-config.js

```diff
@@ -66,6 +66,17 @@ assert.throws(() => {
 }, { code: 'ERR_INVALID_ARG_TYPE' });
 assert.strictEqual(process.report.compact, true);
 
+// Verify that process.report.reportDisableNetwork behaves properly.
+assert.strictEqual(process.report.disableNetwork, true);
+process.report.disableNetwork = false;
```

Comment:
disableNetwork hints me towards a function but this is a property. Should we just call this:

```suggestion
process.report.networkDisabled = false;
```